sdk/servicebus/tests.data.yml

```diff
@@ -7,5 +7,5 @@ extends:
     ServiceDirectory: servicebus
     SDKType: data
     TimeoutInMinutes: 240
-    Clouds: 'Public,Canary'
+    Clouds: 'Public,Preview,UsGov,China,Canary'
```

Comment:
After conversations with @benbp about the intent, we'll want to leave `Clouds` as-is and just ensure that `SupportedClouds` reflect the full set.   We want to limit sovereign clouds to just the weekly runs, not include them in the nightlies or manual runs.